crates/lsp/src/line_index.rs

```diff
@@ -4,17 +4,9 @@
 // origin = "https://github.com/rust-lang/rust-analyzer/blob/master/crates/rust-analyzer/src/line_index.rs"
 // ---
 
-//! Enhances `ide::LineIndex` with additional info required to convert offsets
-//! into lsp positions.
-
-use settings::LineEnding;
 use triomphe::Arc;
 
-use crate::proto::PositionEncoding;
-
 #[derive(Debug, Clone)]
 pub struct LineIndex {
     pub index: Arc<biome_line_index::LineIndex>,
```

Comment:
I'd like to argue that we should drop this wrapper entirely at this point. We currently have no reason to wrap this in an `Arc`. I know rust-analyzer does this, but I think we should simplify things until we start to use salsa, and let the implementation built around salsa guide whether or not we should arc this.